frontend/public/locales/fr/apply-adult-child.json

```
@@ -364,7 +364,7 @@
   },
   "verify-email": {
     "page-title": "Confirmation de votre adresse courriel",
-    "verification-code": "Un courriel contenant un code de vérification a été envoyé à {{email}}, veuillez saisir le code ci-dessous et cliquer sur «\u00a0Continuer\u00a0».",
+    "verification-code": "Un courriel contenant un code de vérification a été envoyé à {{email}}, veuillez saisir le code à 5 chiffres ci-dessous et cliquer sur «\u00a0Continuer\u00a0».",
```

Should be a new sentence according to Figma (same comment for the rest of the files in this PR).

```suggestion
    "verification-code": "Un courriel contenant un code de vérification a été envoyé à {{email}}. Veuillez saisir le code à 5 chiffres ci-dessous et cliquer sur «\u00a0Continuer\u00a0».",
```